superset-frontend/src/explore/components/controls/DateFilterControl/DateFilterControl.tsx

```diff
@@ -168,13 +168,20 @@ const IconWrapper = styled.span`
 interface DateFilterLabelProps {
   name: string;
   onChange: (timeRange: string) => void;
+  onTimeRangeChange?: (textValue?: string, timeRange?: string) => void;
```

Comment:
`onChange` and `onTimeRangeChange` seem to have the same function, can we merge these props?